packages/event-handler/src/rest/BaseRouter.ts

```diff
@@ -0,0 +1,155 @@
+import type { GenericLogger } from '@aws-lambda-powertools/commons/types';
+import { isRecord } from '@aws-lambda-powertools/commons/typeutils';
+import {
+  getStringFromEnv,
+  isDevMode,
+} from '@aws-lambda-powertools/commons/utils/env';
+import type { Context } from 'aws-lambda';
+import type { ResolveOptions } from '../types/index.js';
+import type {
+  RouteHandler,
+  RouteOptions,
+  RouterOptions,
+} from '../types/rest.js';
+
+const HttpVerbs = [
```

Comment:
Since TS is about typing and hints, please add `CONNECT` and `TRACE` methods here, so we have the full implementation on HTTP VERBS. I'll add this in Python.

Reply:
I actually had them in originally but removed them. Will add them back.

Reply:
Done! I also noticed that I'd forgotten the `OPTIONS` method too.